core/conversion/converters/converter_util.cpp

```diff
@@ -0,0 +1,53 @@
+#include "core/conversion/converters/converter_util.h"
+#include "core/conversion/converters/converters.h"
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace conversion {
+namespace converters {
+
+nvinfer1::ITensor* addPadding(ConversionCtx* ctx, const torch::jit::Node* n, nvinfer1::ITensor* tensor, int nDim, bool trailing, bool use_zeros) {
+  const auto dims = tensor->getDimensions();
+
+  if (dims.nbDims < nDim) {
+    auto newDims = dims;
+    for (int dim = dims.nbDims; dim < nDim; ++dim) {
+      newDims = util::unsqueezeDims(newDims, trailing ? dim : 0, 1, use_zeros);
+    }
+
+    LOG_DEBUG("Original shape: " << dims << ", reshaping to: " << newDims);
+    auto shuffle_layer = ctx->net->addShuffle(*tensor);
+    TRTORCH_CHECK(shuffle_layer, "Unable to create shuffle layer");
+    shuffle_layer->setReshapeDimensions(newDims);
+    shuffle_layer->setZeroIsPlaceholder(use_zeros);
+    shuffle_layer->setName((util::node_info(n) + " [Reshape to " + util::toStr(newDims) + ']').c_str());
+    return shuffle_layer->getOutput(0);
+  } else {
+    return nullptr;
```

Comment:
I think that we should either pass through the original tensor here (`tensor`) or error out instead of `nullptr`. What do you think makes the most sense?

Reply:
Looks like we pass through below, we should make them consistent then.

Reply:
Fixed.